javascript-data-model/order-history.js

```diff
@@ -0,0 +1,62 @@
+var orderHistory = [
+  {
+    orderNumber: '114 - 3941689 - 8772232',
+    orderPlaced: 'Aug 4, 2020',
+    delivered: 'Aug 8, 2020',
+    shipTo: 'JS Masher',
+    items: {
+      title: 'JavaScript for Impatient programmers',
+      author: 'Rauschmayer, Dr. Axel',
+      returnWindowClosed: 'Sep 7, 2020',
+      price: '$31.55'
+    },
+    total: '$34.00'
+  },
+  {
+    orderNumber: '112 - 9984268 - 1280257',
+    orderPlaced: 'Jul 19, 2020',
+    delivered: 'Jul 20, 2020',
+    shipTo: 'JS Masher',
+    items: {
+      title: 'The Timeless Way of Building',
+      author: 'Alexander, Christopher',
+      returnWindowClosed: 'Aug 19, 2020',
+      price: '$41.33'
+    },
+    total: '$44.53'
+  },
+  {
+    orderNumber: '114 - 2875557 - 9059409',
+    orderPlaced: 'Jul 4, 2020',
+    delivered: 'Jul 7, 2020',
+    shipTo: 'JS Masher',
+    items: {
+      title: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U...',
+      author: null,
+      returnWindowClosed: 'Aug 5, 2020',
+      price: '$15.98'
+    },
+    total: '$17.22'
+  },
+  {
+    orderNumber: '113 - 2883177 - 2648248',
+    orderPlaced: 'Jul 3, 2020',
+    delivered: 'Jul 5, 2020',
+    shipTo: 'JS Masher',
+    items: [{
```

Comment:
`items` holds an array for this order, but its value is an object in all other orders. That is not consistent and will lead to errors. You should make all of your orders use an array as the value for `items` to create consistency.